docs/application/native/guides/app-management/data-control.md

```diff
@@ -1023,14 +1026,19 @@ The data model must be opened to the public to help other applications to use th
 
 - Data accessibility
 
-  Tizen native applications can control read and write access from other applications by defining data control accessibility.
+  - Tizen native applications can control read and write access from other applications by defining data control accessibility.
+- Trusted
+  - You can allow access from other applications signed with the same certificate by setting the trusted status for the data control.
+- Privileges
+  - Your provider application can restrict access to applications that have specific privileges defined.
```

Comment:
Your provider application can restrict access to applications that have specific privileges defined.

Please rephrase to:

Your provider application can restrict access to applications having certain predefined privileges.